spec/index.html

```diff
@@ -318,29 +375,38 @@ <h2>Introduction</h2>
       used to represent <code>http://purl.org/dc/terms/title</code>.
     </p>
   </section>
+  </section>
+
+<section id="conformance">
+  <p>A <a>YAML-LD document</a> complies with this specification if it follows
+    the normative statements from this specification and
+    can be interpreted as [[JSON-LD]] after transformation into the <a>internal representation</a>.
+    For convenience, normative
+    statements for documents are often phrased as statements on the properties of the document.</p>
+</section>
 
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
     <p>
-      To ease writing and collaborating on JSON-LD documents, it is a common practice
-      to serialize them as YAML.
+      To ease writing and collaborating on [[JSON-LD]] documents, it is a common practice
+      to serialize them as [[YAML]].
       This requires a registered media type, not only to enable content negotiation
       of linked data documents in YAML, but also to define the expected behavior of
       applications that process these documents, including fragment identifiers and
       interoperability considerations.
     </p>
 
     <p>
-      This is because YAML is more flexible than JSON:
+      This is because YAML is more flexible than [[JSON]]:
     </p>
 
     <ul>
-      <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
-      <li>YAML supports more native data types than JSON.</li>
+      <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32;</li>
+      <li>YAML supports more native data types than JSON;</li>
       <li>the structure of a YAML document &mdash; that is, a named YAML representation graph &mdash;
-      is a rooted, directed graph that can have cycles.</li>
-      <li>YAML has the concept of stream, which is a sequence of documents.
+      is a rooted, directed graph that can have cycles;</li>
+      <li>YAML has the concept of a <a>YAML stream</a>, which is a sequence of documents.
           While a stream usually contains one document,
           streams with multiple documents are used to aggregate multiple,
           related, documents into a single file or network stream.
```

Comment:
```suggestion
      <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
      <li>YAML supports more native data types than JSON.</li>
      <li>the structure of a YAML document &mdash; that is, a named YAML representation graph &mdash;
      is a rooted, directed graph that can have cycles.</li>
      <li>YAML has the concept of a <a>YAML stream</a>, which is a sequence of documents.
          While a stream often contains one document,
          streams with multiple documents can be used to aggregate multiple,
          related, documents into a single file or network stream.
```

Reply:
It's not allowing me to commit this suggestion; I'll do it via direct push.